Project Requirements Document.md

```diff
@@ -0,0 +1,28 @@
+﻿
+|    **PROJECT REQUIREMENTS DOCUMENT**   |                                                                                                                                                                                                              |
+|:--------------------------------------:| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|            **Project Name**            | Threewide                                                                                                                                                                                                    |
+|       **Functional Requirements**      |                                                                                                                                                                                                              |
+|            **High Priority**           | 1. Beginners can play Tetris game with modern tetris ruleset.                                                                                                                                                |
+|                                        | 2. Beginners can learn strategies to apply into game.                                                                                                                                                            |
+|                                        | 3. Our goal is to teach number of T spins through our plateform.                                                                                                                                             |
+|                                        | 4. Our plateform should provide some embedded small game tasks to play.                                                                                                                                      |
+|                                        | 5. We should be able to verify that the user accomplished that goal.                                                                                                                                         |
+|            **Low Priority**            | 1. Add more customization to the Tetris game on our plateform for beginners. For instance, we can add teaching strategies for other Tetris blocks such as super rotation system kick table, DAS and ARP etc. |
```

Comment:
Super rotation system is actually part of the high priority. It's central to many T spin set ups. DAS and ARR  are also part of the modern ruleset